### ISSUE TITLE
Hover tool tip not correct

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#22]
If you hover with the mouse in the diagram, there are some elements that show as a tooltip "\n" and others that show "TABLE" instead of valuable information for the user.

**Comment by Neriderc** [comment]
Adjusting this is a bit tricky as ultimately the display is handled by the Viz,js library, but I have managed to influence it.

I notice "<TABLE>" is shown when you enable URLs and hover over the name (which is linked as a URL). Hovering anywhere else on an individual's record shows me the XREF. Disabling URLs means if I hover over the name it shows the XREF as well.

For "\n", I see this when hovering over a family record with no marriage details. If I hover over one with marriage details, it shows me the same thing as written on the record, with a stray "\n" at the end.

I'm not sure how much control we have to influence it. I have succeeded with showing my own text when hovering over the URL but the XREF seems to show at other times because it's the name of the node in the DOT file. It doesn't seem like we can get multiple words in. Perhaps the best thing is just to change all of the hover text to being the XREF of the record?

What were your ideas regarding this?
